### ISSUE TITLE
Unable to replicate a RoleBinding containing a ClusterRole role reference.

### ISSUE THREAD

**Issue by vsimon** [mittwald/kubernetes-replicator#121]
**Describe the bug**
Unable to replicate a RoleBinding containing a ClusterRole role reference.

**To Reproduce**
Steps to reproduce the behavior. Please provide appropriate Kubernetes manifests for reproducing the behavior.

* Create any ClusterRole named  "existing-cluster-role". ClusterRoles exist outside of any namespaces.
* Create namespaces "test-source", "test-destination"
* Apply this RoleBinding in the "test-source" namespace
* 
```
kind: RoleBinding  
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: test-role-binding
  namespace: test-source
  annotations:
    replicator.v1.mittwald.de/replicate-to: test-destination
subjects:
- kind: User
  name: some_user
roleRef:
  kind: ClusterRole
  name: existing-cluster-role
  apiGroup: rbac.authorization.k8s.io
---
```

**Expected behavior**

RoleBinding is replicated to the "test-destination" namespace

**Environment:**
 - Kubernetes version: 1.19
 - kubernetes-replicator version: 2.6.2

**Additional context**

**Comment by vsimon** [comment]
FIrst thought is was to try to restrict the canValidate check to the "Role" kind since it internally calls a "get" only on `r.Client.RbacV1().Roles()`

**Comment by hensur** [comment]
Thank you for the contribution!
Released as `v2.6.3` :)
